src/mgr/DaemonServer.cc

```diff
@@ -1181,6 +1257,12 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  // Track non-admin mgr ops only
```

Comment:
@rzarzynski @linuxkidd If we track the tell or admin socket commands then mgr crashes due to these commands getting handled asynchronously and the `cmdctx->m_tell` using the destroyed connection handle during command execution. We now only intend to track non-admin mgr ops.